docs/index.md

```diff
@@ -31,7 +31,7 @@ terraform {
   required_providers {
     zitadel = {
       source  = "zitadel/zitadel"
-      version = "1.0.0-alpha.16"
+      version = "1.0.7"
```

Comment:
It's a feature. Plz also change the PR title.

Reply:
changed to feat and new version after release